railties/lib/rails/generators/app_base.rb

```diff
@@ -340,7 +340,15 @@ def spring_install?
       end
 
       def run_bundle
-        bundle_command('install') if bundle_install?
+        # Most of us have the CPUs; so let's parallelize `bundle install`
```

Comment:
@lanks read https://github.com/bundler/bundler/pull/3039#issuecomment-43584916 and https://github.com/bundler/bundler/pull/3039#issuecomment-47317208. I don't think you are benchmarking properly.